yfquotes@thegli/files/yfquotes@thegli/po/de.po

```diff
@@ -1,13 +1,13 @@
 # German translation for Yahoo Finance Quotes Desklet
-# Copyright (C) 2021 Thomas Egli
+# Copyright (C) 2023 Thomas Egli
```

Comment:
Just a comment, you normally want this to be the year that the code originated, not when it was updated. You can use a range too if you'd like but that's just extra work.

Reply:
@rcalixte Thanks for the hint. And highly appreciate the super-fast approval :100: